README.md

```diff
@@ -84,30 +84,31 @@ the name of the repository you are configuring homu for.
    config values. 
 
 2. Create a GitHub account that will be used by Homu. You can also use an
-   existing account. In the [account settings][settings], go to "OAuth
-   applications" and create a new application:
+   existing account. In the [developer settings][settings/apps], go to "OAuth
```

Comment:
This is not actually the right place to change the link's target: `settings` refers to line 131, where the full URL is written. You can check in [your fork](https://github.com/emedvedev/homu/blob/patch-1/README.md) whether the links render properly.

Btw you can also fix line 71 since homu is now under the rust-lang org,

Reply:
@kellda all done :)